portfolio/src/main/webapp/script.js

```diff
@@ -36,10 +36,24 @@ function addRandomGreeting() {
 //     });
 // }
 
+function onLoad() {
+  getComments();
+  getLogin();
```

Comment:
You do not need to worry about this now. But you may want give the following comment a thought later.
Do we really need to request the HTML for login every time? Could we make it a part of our HTML file?
You may want to explore AJAX a bit for this too.